Mage/src/main/java/mage/cards/ExpansionSet.java

```diff
@@ -171,7 +175,36 @@ public String toString() {
 
         return theBooster;
     }
+    
+    protected int PartnerCheck(List<Card> booster, boolean partnerAllowed, int max, int i){
+
+        for (Ability ability:booster.get(booster.size() - 1).getAbilities()){
+                    //Check if fetched card has the PartnerWithAbility
+                    if (ability instanceof PartnerWithAbility) {               
+                        //Check if the pack already contains a partner pair
+                        if (partnerAllowed){
+                            //Added card always replaces a common card
```

Comment:
I'm guessing this word should be "uncommon"?

Reply:
Ahh yes sorry for this, let me fix it